src/libstore/binary-cache-store.hh

```diff
@@ -147,7 +148,7 @@ public:
 
     void narFromPath(const StorePath & path, Sink & sink) override;
 
-    ref<SourceAccessor> getFSAccessor(bool requireValidPath) override;
+    ref<SourceAccessor> getFSAccessor(bool requireValidPath = true) override;
```

Comment:
Specifying defaults on overrides should be superfluous, so I think it's best to omit them.

Reply:
Actually see https://en.cppreference.com/w/cpp/language/default_arguments

> The overriders of [virtual](https://en.cppreference.com/w/cpp/language/virtual) functions do not acquire the default arguments from the base class declarations, and when the virtual function call is made, the default arguments are decided based on the static type of the object.